apps/wallet/src/background/db.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import Dexie, { type Table } from 'dexie';
+import { exportDB, importDB } from 'dexie-export-import';
+import { type AccountSourceSerialized } from './account-sources/AccountSource';
+import { type SerializedAccount } from './accounts/Account';
+import { captureException } from './sentry';
+import { getFromLocalStorage, setToLocalStorage } from './storage-utils';
+
+const dbName = 'SuiWallet DB';
+const dbLocalStorageBackupKey = 'indexed-db-backup';
+
+class DB extends Dexie {
+	accountSources!: Table<AccountSourceSerialized, string>;
+	accounts!: Table<SerializedAccount, string>;
+	settings!: Table<{ value: boolean; setting: string }, string>;
+
+	constructor() {
+		super(dbName);
+		this.version(1).stores({
+			accountSources: 'id, type',
+			accounts: 'id, type, address, sourceID',
+			settings: 'setting',
+		});
+	}
+}
+
+async function init() {
+	const db = new DB();
```

Comment:
I updated the way we import the DB from a backup to account for different schema versions. Now we can import a backup the has a previous version that the current, and in case there are any upgrades they will run as expected.